spring-data-geode/src/test/java/org/springframework/data/gemfire/repository/support/SimpleGemfireRepositoryIntegrationTests.java

```diff
@@ -15,24 +15,14 @@
  */
 package org.springframework.data.gemfire.repository.support;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
We should verify where this formatting change comes from.

Reply:
Another case of not configured formatter. Fixed.